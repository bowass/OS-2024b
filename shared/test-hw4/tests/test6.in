51710
34961
49674
16850
29145
39707
52161
31859
56182
13080
12983
28479
60072
25195
31885
27968
4731
24660
12252
51176
42617
46241
30205
59922
38956
48881
10887
2895
63990
9205
21253
41623
47864
50282
58666
60397
31919
31158
21978
23499
46454
54704
47520
53752
20490
53528
22415
23582
31038
55423
42501
18500
40526
48496
52248
49184
43684
13226
27881
451
37651
17060
59645
29926
45911
10068
43765
63223
45886
54625
51838
22646
42247
15053
30702
56493
21382
11187
5978
63411
19578
38769
29195
12831
32165
15155
38992
24111
45688
26387
45228
31429
53832
10960
42808
54786
2607
18889
604
41659
24001
38714
22112
313
12557
11944
36316
126
18846
55273
19450
48334
61872
62142
26986
21173
53877
51105
9801
42315
40519
5857
58173
30422
38374
45469
28256
32922
33836
40352
15078
62127
16035
54656
29421
45957
37529
53937
54064
45241
47972
2271
2037
46007
20637
38663
31775
30188
8111
61952
46721
33269
39630
61693
27352
13941
61347
39454
35873
15517
5747
12632
31957
18554
137
47863
30730
11416
57579
26227
56667
65532
22187
45368
47531
45598
49688
62364
62104
59755
13446
22479
30157
20665
59961
765
51811
23181
3225
15828
57394
57186
58301
6569
2136
61711
28216
64567
15246
28348
28981
60534
49897
45597
44194
57598
31790
59481
46410
35035
117
41167
33752
5394
27945
27638
44174
18515
46203
45183
49291
25465
53482
59386
36811
14013
35640
37472
13417
4619
32976
58511
12453
23607
56152
4747
62439
61243
54871
23101
46874
36821
41352
10003
3237
46794
55604
20063
37796
131
42367
12241
50350
31124
24293
14934
36831
22892
21524
16952
21673
55326
54668
26208
59929
23473
47180
65071
12302
16279
10574
61600
24073
47801
32867
25329
39581
39966
17795
49869
33404
15904
9498
14675
65506
23049
39593
4717
23411
46348
55666
49557
222
57423
35171
30260
22094
48322
22609
45021
16090
58282
29143
20125
2648
3561
18545
42283
25649
52962
59141
13459
23613
934
35030
4433
2302
65136
10168
42079
30810
39987
21223
27567
32561
19413
16101
25849
17052
10717
30821
16448
24877
35111
22690
62579
7259
65023
40764
42652
26714
16905
50988
56994
44861
18593
43221
19945
18619
42946
3250
36142
25309
10988
33634
4240
24910
31950
53622
57899
2253
19441
11380
17797
15573
1381
37956
59498
44187
19059
48695
5089
57453
29692
57355
15292
53548
47245
20246
11287
9444
65228
1417
10816
38683
40365
62184
30086
10806
26009
21431
14631
15667
11892
41095
55474
392
38605
60314
16063
60303
37197
62829
19807
11914
42996
55772
56140
31786
31839
7487
50470
25232
61887
44784
40189
2155
49317
7189
26817
50878
43543
49848
9108
42200
58094
50724
3646
35928
52200
21496
32019
41781
35700
46620
6845
32071
6015
2892
232
8290
3644
7305
27367
61307
46487
387
41927
2665
44398
37135
8689
49210
3077
64919
59309
23851
40423
35628
25779
31036
61446
63297
50650
16777
3270
18953
45805
35878
38092
11158
40914
34761
45393
32424
56484
32906
41066
3068
54427
5986
54526
35674
25353
4267
25553
16721
37542
62501
1008
22193
60454
35513
60867
42732
4200
51094
8202
28442
32824