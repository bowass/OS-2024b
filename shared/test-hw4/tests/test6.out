Virtual address: 51710 Physical address: 254 Value: 50
Virtual address: 34961 Physical address: 401 Value: 0
Virtual address: 49674 Physical address: 522 Value: 48
Virtual address: 16850 Physical address: 978 Value: 16
Virtual address: 29145 Physical address: 1241 Value: 0
Virtual address: 39707 Physical address: 1307 Value: -58
Virtual address: 52161 Physical address: 1729 Value: 0
Virtual address: 31859 Physical address: 1907 Value: 28
Virtual address: 56182 Physical address: 2166 Value: 54
Virtual address: 13080 Physical address: 2328 Value: 0
Virtual address: 12983 Physical address: 2743 Value: -83
Virtual address: 28479 Physical address: 2879 Value: -49
Virtual address: 60072 Physical address: 3240 Value: 0
Virtual address: 25195 Physical address: 3435 Value: -102
Virtual address: 31885 Physical address: 1933 Value: 0
Virtual address: 27968 Physical address: 3648 Value: 0
Virtual address: 4731 Physical address: 3963 Value: -98
Virtual address: 24660 Physical address: 4180 Value: 0
Virtual address: 12252 Physical address: 4572 Value: 0
Virtual address: 51176 Physical address: 4840 Value: 0
Virtual address: 42617 Physical address: 4985 Value: 0
Virtual address: 46241 Physical address: 5281 Value: 0
Virtual address: 30205 Physical address: 5629 Value: 0
Virtual address: 59922 Physical address: 3090 Value: 58
Virtual address: 38956 Physical address: 5676 Value: 0
Virtual address: 48881 Physical address: 6129 Value: 0
Virtual address: 10887 Physical address: 6279 Value: -95
Virtual address: 2895 Physical address: 6479 Value: -45
Virtual address: 63990 Physical address: 6902 Value: 62
Virtual address: 9205 Physical address: 7157 Value: 0
Virtual address: 21253 Physical address: 7173 Value: 0
Virtual address: 41623 Physical address: 7575 Value: -91
Virtual address: 47864 Physical address: 7928 Value: 0
Virtual address: 50282 Physical address: 8042 Value: 49
Virtual address: 58666 Physical address: 8234 Value: 57
Virtual address: 60397 Physical address: 8685 Value: 0
Virtual address: 31919 Physical address: 1967 Value: 43
Virtual address: 31158 Physical address: 8886 Value: 30
Virtual address: 21978 Physical address: 9178 Value: 21
Virtual address: 23499 Physical address: 9419 Value: -14
Virtual address: 46454 Physical address: 9590 Value: 45
Virtual address: 54704 Physical address: 9904 Value: 0
Virtual address: 47520 Physical address: 10144 Value: 0
Virtual address: 53752 Physical address: 10488 Value: 0
Virtual address: 20490 Physical address: 10506 Value: 20
Virtual address: 53528 Physical address: 10264 Value: 0
Virtual address: 22415 Physical address: 10895 Value: -29
Virtual address: 23582 Physical address: 11038 Value: 23
Virtual address: 31038 Physical address: 8766 Value: 30
Virtual address: 55423 Physical address: 11391 Value: 31
Virtual address: 42501 Physical address: 4869 Value: 0
Virtual address: 18500 Physical address: 11588 Value: 0
Virtual address: 40526 Physical address: 11854 Value: 39
Virtual address: 48496 Physical address: 12144 Value: 0
Virtual address: 52248 Physical address: 12312 Value: 0
Virtual address: 49184 Physical address: 12576 Value: 0
Virtual address: 43684 Physical address: 12964 Value: 0
Virtual address: 13226 Physical address: 2474 Value: 12
Virtual address: 27881 Physical address: 13289 Value: 0
Virtual address: 451 Physical address: 13507 Value: 112
Virtual address: 37651 Physical address: 13587 Value: -60
Virtual address: 17060 Physical address: 13988 Value: 0
Virtual address: 59645 Physical address: 14333 Value: 0
Virtual address: 29926 Physical address: 14566 Value: 29
Virtual address: 45911 Physical address: 14679 Value: -43
Virtual address: 10068 Physical address: 14932 Value: 0
Virtual address: 43765 Physical address: 13045 Value: 0
Virtual address: 63223 Physical address: 15351 Value: -67
Virtual address: 45886 Physical address: 14654 Value: 44
Virtual address: 54625 Physical address: 9825 Value: 0
Virtual address: 51838 Physical address: 15486 Value: 50
Virtual address: 22646 Physical address: 15734 Value: 22
Virtual address: 42247 Physical address: 15879 Value: 65
Virtual address: 15053 Physical address: 16333 Value: 0
Virtual address: 30702 Physical address: 16622 Value: 29
Virtual address: 56493 Physical address: 16813 Value: 0
Virtual address: 21382 Physical address: 7302 Value: 20
Virtual address: 11187 Physical address: 17075 Value: -20
Virtual address: 5978 Physical address: 17242 Value: 5
Virtual address: 63411 Physical address: 17587 Value: -20
Virtual address: 19578 Physical address: 17786 Value: 19
Virtual address: 38769 Physical address: 18033 Value: 0
Virtual address: 29195 Physical address: 18187 Value: -126
Virtual address: 12831 Physical address: 2591 Value: -121
Virtual address: 32165 Physical address: 18597 Value: 0
Virtual address: 15155 Physical address: 18739 Value: -52
Virtual address: 38992 Physical address: 5712 Value: 0
Virtual address: 24111 Physical address: 18991 Value: -117
Virtual address: 45688 Physical address: 19320 Value: 0
Virtual address: 26387 Physical address: 19475 Value: -60
Virtual address: 45228 Physical address: 19884 Value: 0
Virtual address: 31429 Physical address: 20165 Value: 0
Virtual address: 53832 Physical address: 20296 Value: 0
Virtual address: 10960 Physical address: 6352 Value: 0
Virtual address: 42808 Physical address: 20536 Value: 0
Virtual address: 54786 Physical address: 20738 Value: 53
Virtual address: 2607 Physical address: 21039 Value: -117
Virtual address: 18889 Physical address: 21449 Value: 0
Virtual address: 604 Physical address: 21596 Value: 0
Virtual address: 41659 Physical address: 7611 Value: -82
Virtual address: 24001 Physical address: 21953 Value: 0
Virtual address: 38714 Physical address: 17978 Value: 37
Virtual address: 22112 Physical address: 22112 Value: 0
Virtual address: 313 Physical address: 13369 Value: 0
Virtual address: 12557 Physical address: 22285 Value: 0
Virtual address: 11944 Physical address: 22696 Value: 0
Virtual address: 36316 Physical address: 23004 Value: 0
Virtual address: 126 Physical address: 23166 Value: 0
Virtual address: 18846 Physical address: 21406 Value: 18
Virtual address: 55273 Physical address: 23529 Value: 0
Virtual address: 19450 Physical address: 23802 Value: 18
Virtual address: 48334 Physical address: 24014 Value: 47
Virtual address: 61872 Physical address: 24240 Value: 0
Virtual address: 62142 Physical address: 24510 Value: 60
Virtual address: 26986 Physical address: 24682 Value: 26
Virtual address: 21173 Physical address: 25013 Value: 0
Virtual address: 53877 Physical address: 20341 Value: 0
Virtual address: 51105 Physical address: 4769 Value: 0
Virtual address: 9801 Physical address: 25161 Value: 0
Virtual address: 42315 Physical address: 15947 Value: 82
Virtual address: 40519 Physical address: 11847 Value: -111
Virtual address: 5857 Physical address: 25569 Value: 0
Virtual address: 58173 Physical address: 25661 Value: 0
Virtual address: 30422 Physical address: 26070 Value: 29
Virtual address: 38374 Physical address: 26342 Value: 37
Virtual address: 45469 Physical address: 26525 Value: 0
Virtual address: 28256 Physical address: 26720 Value: 0
Virtual address: 32922 Physical address: 27034 Value: 32
Virtual address: 33836 Physical address: 27180 Value: 0
Virtual address: 40352 Physical address: 27552 Value: 0
Virtual address: 15078 Physical address: 16358 Value: 14
Virtual address: 62127 Physical address: 24495 Value: -85
Virtual address: 16035 Physical address: 27811 Value: -88
Virtual address: 54656 Physical address: 9856 Value: 0
Virtual address: 29421 Physical address: 18413 Value: 0
Virtual address: 45957 Physical address: 14725 Value: 0
Virtual address: 37529 Physical address: 28057 Value: 0
Virtual address: 53937 Physical address: 20401 Value: 0
Virtual address: 54064 Physical address: 28208 Value: 0
Virtual address: 45241 Physical address: 19897 Value: 0
Virtual address: 47972 Physical address: 28516 Value: 0
Virtual address: 2271 Physical address: 28895 Value: 55
Virtual address: 2037 Physical address: 29173 Value: 0
Virtual address: 46007 Physical address: 14775 Value: -19
Virtual address: 20637 Physical address: 10653 Value: 0
Virtual address: 38663 Physical address: 17927 Value: -63
Virtual address: 31775 Physical address: 1823 Value: 7
Virtual address: 30188 Physical address: 5612 Value: 0
Virtual address: 8111 Physical address: 29359 Value: -21
Virtual address: 61952 Physical address: 24320 Value: 0
Virtual address: 46721 Physical address: 29569 Value: 0
Virtual address: 33269 Physical address: 29941 Value: 0
Virtual address: 39630 Physical address: 30158 Value: 38
Virtual address: 61693 Physical address: 30461 Value: 0
Virtual address: 27352 Physical address: 30680 Value: 0
Virtual address: 13941 Physical address: 30837 Value: 0
Virtual address: 61347 Physical address: 31139 Value: -24
Virtual address: 39454 Physical address: 29982 Value: 38
Virtual address: 35873 Physical address: 31265 Value: 0
Virtual address: 15517 Physical address: 31645 Value: 0
Virtual address: 5747 Physical address: 25459 Value: -100
Virtual address: 12632 Physical address: 22360 Value: 0
Virtual address: 31957 Physical address: 2005 Value: 0
Virtual address: 18554 Physical address: 11642 Value: 18
Virtual address: 137 Physical address: 23177 Value: 0
Virtual address: 47863 Physical address: 7927 Value: -67
Virtual address: 30730 Physical address: 31754 Value: 30
Virtual address: 11416 Physical address: 32152 Value: 0
Virtual address: 57579 Physical address: 32491 Value: 58
Virtual address: 26227 Physical address: 32627 Value: -100
Virtual address: 56667 Physical address: 32859 Value: 86
Virtual address: 65532 Physical address: 33276 Value: 0
Virtual address: 22187 Physical address: 22187 Value: -86
Virtual address: 45368 Physical address: 26424 Value: 0
Virtual address: 47531 Physical address: 10155 Value: 106
Virtual address: 45598 Physical address: 19230 Value: 44
Virtual address: 49688 Physical address: 536 Value: 0
Virtual address: 62364 Physical address: 33436 Value: 0
Virtual address: 62104 Physical address: 24472 Value: 0
Virtual address: 59755 Physical address: 33643 Value: 90
Virtual address: 13446 Physical address: 33926 Value: 13
Virtual address: 22479 Physical address: 10959 Value: -13
Virtual address: 30157 Physical address: 5581 Value: 0
Virtual address: 20665 Physical address: 10681 Value: 0
Virtual address: 59961 Physical address: 3129 Value: 0
Virtual address: 765 Physical address: 21757 Value: 0
Virtual address: 51811 Physical address: 15459 Value: -104
Virtual address: 23181 Physical address: 34189 Value: 0
Virtual address: 3225 Physical address: 34457 Value: 0
Virtual address: 15828 Physical address: 34772 Value: 0
Virtual address: 57394 Physical address: 32306 Value: 56
Virtual address: 57186 Physical address: 34914 Value: 55
Virtual address: 58301 Physical address: 25789 Value: 0
Virtual address: 6569 Physical address: 35241 Value: 0
Virtual address: 2136 Physical address: 28760 Value: 0
Virtual address: 61711 Physical address: 24079 Value: 67
Virtual address: 28216 Physical address: 26680 Value: 0
Virtual address: 64567 Physical address: 35383 Value: 13
Virtual address: 15246 Physical address: 18830 Value: 14
Virtual address: 28348 Physical address: 26812 Value: 0
Virtual address: 28981 Physical address: 1077 Value: 0
Virtual address: 60534 Physical address: 35702 Value: 59
Virtual address: 49897 Physical address: 745 Value: 0
Virtual address: 45597 Physical address: 19229 Value: 0
Virtual address: 44194 Physical address: 36002 Value: 43
Virtual address: 57598 Physical address: 32510 Value: 56
Virtual address: 31790 Physical address: 1838 Value: 31
Virtual address: 59481 Physical address: 14169 Value: 0
Virtual address: 46410 Physical address: 9546 Value: 45
Virtual address: 35035 Physical address: 475 Value: 54
Virtual address: 117 Physical address: 23157 Value: 0
Virtual address: 41167 Physical address: 36303 Value: 51
Virtual address: 33752 Physical address: 36568 Value: 0
Virtual address: 5394 Physical address: 36626 Value: 5
Virtual address: 27945 Physical address: 3625 Value: 0
Virtual address: 27638 Physical address: 37110 Value: 26
Virtual address: 44174 Physical address: 35982 Value: 43
Virtual address: 18515 Physical address: 11603 Value: 20
Virtual address: 46203 Physical address: 5243 Value: 30
Virtual address: 45183 Physical address: 19839 Value: 31
Virtual address: 49291 Physical address: 12683 Value: 34
Virtual address: 25465 Physical address: 37241 Value: 0
Virtual address: 53482 Physical address: 37610 Value: 52
Virtual address: 59386 Physical address: 37882 Value: 57
Virtual address: 36811 Physical address: 38091 Value: -14
Virtual address: 14013 Physical address: 30909 Value: 0
Virtual address: 35640 Physical address: 38200 Value: 0
Virtual address: 37472 Physical address: 28000 Value: 0
Virtual address: 13417 Physical address: 33897 Value: 0
Virtual address: 4619 Physical address: 3851 Value: -126
Virtual address: 32976 Physical address: 27088 Value: 0
Virtual address: 58511 Physical address: 38543 Value: 35
Virtual address: 12453 Physical address: 38821 Value: 0
Virtual address: 23607 Physical address: 11063 Value: 13
Virtual address: 56152 Physical address: 2136 Value: 0
Virtual address: 4747 Physical address: 3979 Value: -94
Virtual address: 62439 Physical address: 33511 Value: -7
Virtual address: 61243 Physical address: 31035 Value: -50
Virtual address: 54871 Physical address: 20823 Value: -107
Virtual address: 23101 Physical address: 34109 Value: 0
Virtual address: 46874 Physical address: 38938 Value: 45
Virtual address: 36821 Physical address: 38101 Value: 0
Virtual address: 41352 Physical address: 39304 Value: 0
Virtual address: 10003 Physical address: 14867 Value: -60
Virtual address: 3237 Physical address: 34469 Value: 0
Virtual address: 46794 Physical address: 29642 Value: 45
Virtual address: 55604 Physical address: 39476 Value: 0
Virtual address: 20063 Physical address: 39775 Value: -105
Virtual address: 37796 Physical address: 13732 Value: 0
Virtual address: 131 Physical address: 23171 Value: 32
Virtual address: 42367 Physical address: 15999 Value: 95
Virtual address: 12241 Physical address: 4561 Value: 0
Virtual address: 50350 Physical address: 8110 Value: 49
Virtual address: 31124 Physical address: 8852 Value: 0
Virtual address: 24293 Physical address: 19173 Value: 0
Virtual address: 14934 Physical address: 16214 Value: 14
Virtual address: 36831 Physical address: 38111 Value: -9
Virtual address: 22892 Physical address: 40044 Value: 0
Virtual address: 21524 Physical address: 40212 Value: 0
Virtual address: 16952 Physical address: 13880 Value: 0
Virtual address: 21673 Physical address: 40361 Value: 0
Virtual address: 55326 Physical address: 11294 Value: 54
Virtual address: 54668 Physical address: 9868 Value: 0
Virtual address: 26208 Physical address: 32608 Value: 0
Virtual address: 59929 Physical address: 3097 Value: 0
Virtual address: 23473 Physical address: 9393 Value: 0
Virtual address: 47180 Physical address: 40524 Value: 0
Virtual address: 65071 Physical address: 40751 Value: -117
Virtual address: 12302 Physical address: 38670 Value: 12
Virtual address: 16279 Physical address: 41111 Value: -27
Virtual address: 10574 Physical address: 41294 Value: 10
Virtual address: 61600 Physical address: 30368 Value: 0
Virtual address: 24073 Physical address: 18953 Value: 0
Virtual address: 47801 Physical address: 7865 Value: 0
Virtual address: 32867 Physical address: 26979 Value: 24
Virtual address: 25329 Physical address: 3569 Value: 0
Virtual address: 39581 Physical address: 30109 Value: 0
Virtual address: 39966 Physical address: 41502 Value: 39
Virtual address: 17795 Physical address: 41859 Value: 96
Virtual address: 49869 Physical address: 717 Value: 0
Virtual address: 33404 Physical address: 42108 Value: 0
Virtual address: 15904 Physical address: 27680 Value: 0
Virtual address: 9498 Physical address: 42266 Value: 9
Virtual address: 14675 Physical address: 42579 Value: 84
Virtual address: 65506 Physical address: 33250 Value: 63
Virtual address: 23049 Physical address: 34057 Value: 0
Virtual address: 39593 Physical address: 30121 Value: 0
Virtual address: 4717 Physical address: 3949 Value: 0
Virtual address: 23411 Physical address: 9331 Value: -36
Virtual address: 46348 Physical address: 9484 Value: 0
Virtual address: 55666 Physical address: 39538 Value: 54
Virtual address: 49557 Physical address: 42901 Value: 0
Virtual address: 222 Physical address: 23262 Value: 0
Virtual address: 57423 Physical address: 32335 Value: 19
Virtual address: 35171 Physical address: 43107 Value: 88
Virtual address: 30260 Physical address: 25908 Value: 0
Virtual address: 22094 Physical address: 22094 Value: 21
Virtual address: 48322 Physical address: 24002 Value: 47
Virtual address: 22609 Physical address: 15697 Value: 0
Virtual address: 45021 Physical address: 43485 Value: 0
Virtual address: 16090 Physical address: 27866 Value: 15
Virtual address: 58282 Physical address: 25770 Value: 56
Virtual address: 29143 Physical address: 1239 Value: 117
Virtual address: 20125 Physical address: 39837 Value: 0
Virtual address: 2648 Physical address: 21080 Value: 0
Virtual address: 3561 Physical address: 43753 Value: 0
Virtual address: 18545 Physical address: 11633 Value: 0
Virtual address: 42283 Physical address: 15915 Value: 74
Virtual address: 25649 Physical address: 43825 Value: 0
Virtual address: 52962 Physical address: 44258 Value: 51
Virtual address: 59141 Physical address: 37637 Value: 0
Virtual address: 13459 Physical address: 33939 Value: 36
Virtual address: 23613 Physical address: 11069 Value: 0
Virtual address: 934 Physical address: 44454 Value: 0
Virtual address: 35030 Physical address: 470 Value: 34
Virtual address: 4433 Physical address: 44625 Value: 0
Virtual address: 2302 Physical address: 28926 Value: 2
Virtual address: 65136 Physical address: 40816 Value: 0
Virtual address: 10168 Physical address: 15032 Value: 0
Virtual address: 42079 Physical address: 44895 Value: 23
Virtual address: 30810 Physical address: 31834 Value: 30
Virtual address: 39987 Physical address: 41523 Value: 12
Virtual address: 21223 Physical address: 25063 Value: -71
Virtual address: 27567 Physical address: 37039 Value: -21
Virtual address: 32561 Physical address: 45105 Value: 0
Virtual address: 19413 Physical address: 23765 Value: 0
Virtual address: 16101 Physical address: 27877 Value: 0
Virtual address: 25849 Physical address: 44025 Value: 0
Virtual address: 17052 Physical address: 13980 Value: 0
Virtual address: 10717 Physical address: 41437 Value: 0
Virtual address: 30821 Physical address: 31845 Value: 0
Virtual address: 16448 Physical address: 45376 Value: 0
Virtual address: 24877 Physical address: 45613 Value: 0
Virtual address: 35111 Physical address: 43047 Value: 73
Virtual address: 22690 Physical address: 15778 Value: 22
Virtual address: 62579 Physical address: 45939 Value: 28
Virtual address: 7259 Physical address: 46171 Value: 22
Virtual address: 65023 Physical address: 46591 Value: 127
Virtual address: 40764 Physical address: 46652 Value: 0
Virtual address: 42652 Physical address: 5020 Value: 0
Virtual address: 26714 Physical address: 46938 Value: 26
Virtual address: 16905 Physical address: 13833 Value: 0
Virtual address: 50988 Physical address: 4652 Value: 0
Virtual address: 56994 Physical address: 47266 Value: 55
Virtual address: 44861 Physical address: 43325 Value: 0
Virtual address: 18593 Physical address: 11681 Value: 0
Virtual address: 43221 Physical address: 47573 Value: 0
Virtual address: 19945 Physical address: 47849 Value: 0
Virtual address: 18619 Physical address: 11707 Value: 46
Virtual address: 42946 Physical address: 20674 Value: 41
Virtual address: 3250 Physical address: 34482 Value: 3
Virtual address: 36142 Physical address: 22830 Value: 35
Virtual address: 25309 Physical address: 3549 Value: 0
Virtual address: 10988 Physical address: 6380 Value: 0
Virtual address: 33634 Physical address: 36450 Value: 32
Virtual address: 4240 Physical address: 48016 Value: 0
Virtual address: 24910 Physical address: 45646 Value: 24
Virtual address: 31950 Physical address: 1998 Value: 31
Virtual address: 53622 Physical address: 10358 Value: 52
Virtual address: 57899 Physical address: 48171 Value: -118
Virtual address: 2253 Physical address: 28877 Value: 0
Virtual address: 19441 Physical address: 23793 Value: 0
Virtual address: 11380 Physical address: 32116 Value: 0
Virtual address: 17797 Physical address: 41861 Value: 0
Virtual address: 15573 Physical address: 31701 Value: 0
Virtual address: 1381 Physical address: 48485 Value: 0
Virtual address: 37956 Physical address: 48708 Value: 0
Virtual address: 59498 Physical address: 14186 Value: 58
Virtual address: 44187 Physical address: 35995 Value: 38
Virtual address: 19059 Physical address: 49011 Value: -100
Virtual address: 48695 Physical address: 5943 Value: -115
Virtual address: 5089 Physical address: 49377 Value: 0
Virtual address: 57453 Physical address: 32365 Value: 0
Virtual address: 29692 Physical address: 49660 Value: 0
Virtual address: 57355 Physical address: 32267 Value: 2
Virtual address: 15292 Physical address: 18876 Value: 0
Virtual address: 53548 Physical address: 10284 Value: 0
Virtual address: 47245 Physical address: 40589 Value: 0
Virtual address: 20246 Physical address: 49686 Value: 19
Virtual address: 11287 Physical address: 32023 Value: 5
Virtual address: 9444 Physical address: 50148 Value: 0
Virtual address: 65228 Physical address: 40908 Value: 0
Virtual address: 1417 Physical address: 48521 Value: 0
Virtual address: 10816 Physical address: 6208 Value: 0
Virtual address: 38683 Physical address: 17947 Value: -58
Virtual address: 40365 Physical address: 27565 Value: 0
Virtual address: 62184 Physical address: 24552 Value: 0
Virtual address: 30086 Physical address: 5510 Value: 29
Virtual address: 10806 Physical address: 6198 Value: 10
Virtual address: 26009 Physical address: 50329 Value: 0
Virtual address: 21431 Physical address: 7351 Value: -19
Virtual address: 14631 Physical address: 42535 Value: 73
Virtual address: 15667 Physical address: 34611 Value: 76
Virtual address: 11892 Physical address: 22644 Value: 0
Virtual address: 41095 Physical address: 36231 Value: 33
Virtual address: 55474 Physical address: 11442 Value: 54
Virtual address: 392 Physical address: 13448 Value: 0
Virtual address: 38605 Physical address: 50637 Value: 0
Virtual address: 60314 Physical address: 8602 Value: 58
Virtual address: 16063 Physical address: 27839 Value: -81
Virtual address: 60303 Physical address: 8591 Value: -29
Virtual address: 37197 Physical address: 50765 Value: 0
Virtual address: 62829 Physical address: 51053 Value: 0
Virtual address: 19807 Physical address: 47711 Value: 87
Virtual address: 11914 Physical address: 22666 Value: 11
Virtual address: 42996 Physical address: 20724 Value: 0
Virtual address: 55772 Physical address: 39644 Value: 0
Virtual address: 56140 Physical address: 2124 Value: 0
Virtual address: 31786 Physical address: 1834 Value: 31
Virtual address: 31839 Physical address: 1887 Value: 23
Virtual address: 7487 Physical address: 51263 Value: 79
Virtual address: 50470 Physical address: 51494 Value: 49
Virtual address: 25232 Physical address: 3472 Value: 0
Virtual address: 61887 Physical address: 24255 Value: 111
Virtual address: 44784 Physical address: 51952 Value: 0
Virtual address: 40189 Physical address: 41725 Value: 0
Virtual address: 2155 Physical address: 28779 Value: 26
Virtual address: 49317 Physical address: 12709 Value: 0
Virtual address: 7189 Physical address: 46101 Value: 0
Virtual address: 26817 Physical address: 47041 Value: 0
Virtual address: 50878 Physical address: 52158 Value: 49
Virtual address: 43543 Physical address: 12823 Value: -123
Virtual address: 49848 Physical address: 696 Value: 0
Virtual address: 9108 Physical address: 7060 Value: 0
Virtual address: 42200 Physical address: 45016 Value: 0
Virtual address: 58094 Physical address: 48366 Value: 56
Virtual address: 50724 Physical address: 52004 Value: 0
Virtual address: 3646 Physical address: 52286 Value: 3
Virtual address: 35928 Physical address: 31320 Value: 0
Virtual address: 52200 Physical address: 1768 Value: 0
Virtual address: 21496 Physical address: 7416 Value: 0
Virtual address: 32019 Physical address: 18451 Value: 68
Virtual address: 41781 Physical address: 52533 Value: 0
Virtual address: 35700 Physical address: 38260 Value: 0
Virtual address: 46620 Physical address: 29468 Value: 0
Virtual address: 6845 Physical address: 52925 Value: 0
Virtual address: 32071 Physical address: 18503 Value: 81
Virtual address: 6015 Physical address: 17279 Value: -33
Virtual address: 2892 Physical address: 6476 Value: 0
Virtual address: 232 Physical address: 23272 Value: 0
Virtual address: 8290 Physical address: 53090 Value: 8
Virtual address: 3644 Physical address: 52284 Value: 0
Virtual address: 7305 Physical address: 46217 Value: 0
Virtual address: 27367 Physical address: 30695 Value: -71
Virtual address: 61307 Physical address: 31099 Value: -34
Virtual address: 46487 Physical address: 9623 Value: 101
Virtual address: 387 Physical address: 13443 Value: 96
Virtual address: 41927 Physical address: 52679 Value: -15
Virtual address: 2665 Physical address: 21097 Value: 0
Virtual address: 44398 Physical address: 53358 Value: 43
Virtual address: 37135 Physical address: 50703 Value: 67
Virtual address: 8689 Physical address: 53745 Value: 0
Virtual address: 49210 Physical address: 12602 Value: 48
Virtual address: 3077 Physical address: 34309 Value: 0
Virtual address: 64919 Physical address: 46487 Value: 101
Virtual address: 59309 Physical address: 37805 Value: 0
Virtual address: 23851 Physical address: 21803 Value: 74
Virtual address: 40423 Physical address: 27623 Value: 121
Virtual address: 35628 Physical address: 38188 Value: 0
Virtual address: 25779 Physical address: 43955 Value: 44
Virtual address: 31036 Physical address: 8764 Value: 0
Virtual address: 61446 Physical address: 30214 Value: 60
Virtual address: 63297 Physical address: 17473 Value: 0
Virtual address: 50650 Physical address: 51674 Value: 49
Virtual address: 16777 Physical address: 905 Value: 0
Virtual address: 3270 Physical address: 34502 Value: 3
Virtual address: 18953 Physical address: 48905 Value: 0
Virtual address: 45805 Physical address: 19437 Value: 0
Virtual address: 35878 Physical address: 31270 Value: 35
Virtual address: 38092 Physical address: 48844 Value: 0
Virtual address: 11158 Physical address: 17046 Value: 10
Virtual address: 40914 Physical address: 46802 Value: 39
Virtual address: 34761 Physical address: 53961 Value: 0
Virtual address: 45393 Physical address: 26449 Value: 0
Virtual address: 32424 Physical address: 54184 Value: 0
Virtual address: 56484 Physical address: 16804 Value: 0
Virtual address: 32906 Physical address: 27018 Value: 32
Virtual address: 41066 Physical address: 36202 Value: 40
Virtual address: 3068 Physical address: 6652 Value: 0
Virtual address: 54427 Physical address: 54427 Value: 38
Virtual address: 5986 Physical address: 17250 Value: 5
Virtual address: 54526 Physical address: 54526 Value: 53
Virtual address: 35674 Physical address: 38234 Value: 34
Virtual address: 25353 Physical address: 37129 Value: 0
Virtual address: 4267 Physical address: 48043 Value: 42
Virtual address: 25553 Physical address: 37329 Value: 0
Virtual address: 16721 Physical address: 849 Value: 0
Virtual address: 37542 Physical address: 28070 Value: 36
Virtual address: 62501 Physical address: 45861 Value: 0
Virtual address: 1008 Physical address: 44528 Value: 0
Virtual address: 22193 Physical address: 22193 Value: 0
Virtual address: 60454 Physical address: 35622 Value: 59
Virtual address: 35513 Physical address: 54713 Value: 0
Virtual address: 60867 Physical address: 54979 Value: 112
Virtual address: 42732 Physical address: 5100 Value: 0
Virtual address: 4200 Physical address: 47976 Value: 0
Virtual address: 51094 Physical address: 4758 Value: 49
Virtual address: 8202 Physical address: 53002 Value: 8
Virtual address: 28442 Physical address: 2842 Value: 27
Virtual address: 32824 Physical address: 26936 Value: 0
Number of Translated Addresses = 500
Page Faults = 215
Page Fault Rate = 0.430
TLB Hits = 35
TLB Hit Rate = 0.070
